src/otx/algo/classification/backbones/efficientnet.py

```diff
@@ -661,5 +661,6 @@ def init_weights(self, pretrained: bool | str | None = None):
             load_checkpoint_to_model(self, checkpoint)
             print(f"init weight - {pretrained}")
         elif pretrained is not None:
-            download_model(net=self, model_name=self.model_name)
+            cache_dir = os.path.join("~", ".cache", "torch", "hub", "checkpoints")
```

Comment:
Could you change a code to use `pathlib`?

Reply:
Thanks - Done https://github.com/openvinotoolkit/training_extensions/pull/3656/commits/af14fabc0051ff7d5c611c81c80a132a0006f602